src/NuGet.Core/NuGet.ProjectModel/NuGet.ProjectModel.csproj

```diff
@@ -45,4 +45,8 @@
   <ItemGroup>
     <InternalsVisibleTo Include="NuGet.ProjectModel.Test" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Text.Json" />
```

Comment:
See my pull request, after it was rebased, following retargeting the projects to net8.0:
* https://github.com/NuGet/NuGet.Client/pull/6083

We shouldn't include a package reference for target frameworks where this assembly is already part of the targeting pack.

Reply:
Please add a condition for the inclusion of the PR.